tasks/12_title-case/title-case.js

```diff
@@ -0,0 +1,7 @@
+export const titleCase = (value) => {
+  let wordsArray = value.toLowerCase().split((/\s/g));
+  let changedArray = wordsArray.map((cur) => {
+    return `${cur.charAt(0).toUpperCase()}${cur.substr(1)}`;
```

Comment:
better to use `String.prototype.substring` as `String.prototype.substr` is kind of legacy [more details here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr)